cfgov/v1/apps.py

```diff
@@ -0,0 +1,13 @@
+from django.apps import AppConfig
+from django.db.models.signals import post_save
+from django.contrib.auth import get_user_model
+
+from .signals import user_save_callback
+
+class V1AppConfig(AppConfig):
```

Comment:
you know what. i think we should probably start getting into the habit of not using `v1` anymore to name stuff since it'll be a pain when we start referring to the project as `v2` maybe we can call this `CFGovAppConfig` and edit the names?